config/reportgenerator.properties

```diff
@@ -47,8 +47,8 @@
 com.xceptance.xlt.reportgenerator.charts.width = 900
 com.xceptance.xlt.reportgenerator.charts.height = 300
 
-## The PNG compression level to use when generating charts ([0..9], default: 6).
-#com.xceptance.xlt.reportgenerator.charts.compressionLevel = 6
+## The Webp compression quality to use when generating charts ([0..1], default: 0.75).
```

Comment:
What about adding info about the impact the value has, e.g. 0 -> fastest, 1 -> best compression?

Reply:
Done